lib/sources/chain/chain_source_impl.dart

```diff
@@ -159,7 +159,13 @@ class ChainSourceImpl extends ChainSource {
     }
 
     final data = TxData(messages: messages, wallet: wallet, feeAmount: fees);
-    return compute(sendTxBackground, data);
+
+    TransactionResult txResults = await compute(sendTxBackground, data);
+    // logs to sentry
+    if (!txResults.success) {
+      Logger.log(txResults.error);
+    }
```

Comment:
Just a small change to make it more readable from Sentry:

```suggestion
    if (!txResults.success) {
      Logger.log("Error while sending transaction to the chain: ${txResults.error.errorMessage}");
    }
```